README.md

```diff
@@ -1,16 +1,254 @@
-**HEADS UP: This project is under active development.** While the beta should be relatively stable, major bugs may still be present--use at your own risk in production environments.
+# Athena Framework
+
+A front-end web framework for UCF websites and applications.  Developed and maintained by [UCF Web Communications](https://www.ucf.edu/brand/contact-us/).
+
+
+## Quick Links
+
+- [**Framework Documentation**](https://ucf.github.io/Athena-Framework/)
+- [Development](#development)
+- [Contributing](#contributing)
+- [License](#license)
 
 ---
 
-# Athena-Framework
-A front-end web framework for UCF websites and applications
+## Development
+
+_These instructions are for installation and development of the project for core development of the Athena Framework. If you're looking for instructions for installing and using the framework in your project, [see our documentation](https://ucf.github.io/Athena-Framework/)._
+
+### Installation requirements
+
+- [Node.js](https://nodejs.org/en/download/)
+- [gulp-cli](https://github.com/gulpjs/gulp/blob/v3.9.1/docs/getting-started.md#1-install-gulp-globally)
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
```

Comment:
Ruby >= 2.2.+ The version requirement needs to be documented. It's not immediately obvious if you're using < 2.2 when the error gets thrown.